client/src/Containers/Addquestion.jsx

```diff
@@ -23,6 +34,62 @@ const Addquestion = () => {
     const option4 = document.getElementById('option4-input').value;
     const correctAnswer = document.getElementById('correct-answer').value;
     const timeGiven = document.getElementById('time-given').value;

```

Comment:
using `document.getElementById` should be avoided when you're using React because it maintains a virtual DOM.

Reply:
Instead, you should bind the value with a state variable. 
It should be something like:
```
const [state, setState] = useState();
...
function changeHandler(e) {
  setState(e.target.value)
}
...
<input onChange={changeHandler} value={state}>
```

Reply:
I have tried to implement some of the changes, kindly review it once and let me know any further changes to be made

Reply:
@KShivendu can you please re-review these changes?